model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -193,65 +193,133 @@ case class EntityCopyDefinition(
                    entityNames: Seq[String]
                    )
 
-// In the next phase (GAWB-3100), this class's members will be swapped out for a singular methodUri
```

Comment:
In hindsight this seems wrong because referencing the fields of case classes is amazing and repeatedly parsing & re-generating a URI string in memory is painful